### DISCUSSION TITLE
Is Target Flow Temperature available through the API?

### DISCUSSION THREAD

**Discussion by jeffjsmith33** [signalkraft/mypyllant-component#211] [Ideas]
I was wondering if target flow temperature might be available as an entity if it is available via the API.  It is available for display with the SensoComfort device.  It would be interesting sometimes to be able to plot this against the outside temperature when tuning the system.  Thanks for the work you have done on the Integration it is really useful!

**Comment by bernd32457** [comment]
Yes, the target temperature is available . Here are my temperature values ​​in Home Assistant.
current_temperature: 19.9
**_temperature: 21.5_**
current_humidity: 74
